src/constants.h

```diff
@@ -108,8 +108,7 @@ enum class PinSubCommand : uint8_t {
   kSetPin = 0x03,
   kChangePin = 0x04,
   kGetPinToken = 0x05,
-  kGetPinUvAuthTokenUsingPin = 0x05,
```

Comment:
I don't even want to know what it means in C++ to have twice the same enum value...

Reply:
It did seem to work, but it was an oversight to have it there twice.